Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -274,12 +342,18 @@ Note: in step 5, if you choose **Use existing app registration**, make sure to d
 
 ## Known Limitations
 ---
-In some cases, you might encounter a problem, where no communication is created between Teams and the messaging endpoint, when adding a bot to the team. You can workaround this problem by adding any member to the team the bot was added to. It's supposed to trigger a communication and solve the issue.
+- In some cases, you might encounter a problem, where no communication is created between Teams and the messaging endpoint, when adding a bot to the team. You can workaround this problem by adding any member to the team the bot was added to. It's supposed to trigger a communication and solve the issue.
```

Comment:
```suggestion
- In some cases, you might encounter a problem, where no communication is created between Teams and the messaging endpoint, when adding a bot to the team. You can work around this problem by adding any member to the team the bot was added to. It will trigger a communication and solve the issue.
```